### PR TITLE
Bump OpenStack provider to 1.1.0

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

File: modules/tls/kube/self-signed/outputs.tf [hunk 1]
```diff
@@ -29,12 +29,12 @@ output "apiserver_key_pem" {
 output "id" {
   value = "${sha1("
   ${join(" ",
-    local_file.apiserver_key.id,
+    list(local_file.apiserver_key.id,
```
Comment (squat) [inline modules/tls/kube/self-signed/outputs.tf]: These changes should go in another PR

**Comment by lblackstone** [comment]
@squat Moved other commit to #2494

**Comment by squat** [comment]
ok to test

**Comment by squat** [comment]
ok to test
